### DISCUSSION TITLE
current vector naming in Nutmeg formula

### DISCUSSION THREAD

**Discussion by bobu01** [ra3xdh/qucs_s#241] [Q&A] [answered]
A Nutmeg formula will not run when a current vector is in `i(device)` format.  A voltage vector in `v(net)` format does work.  A current vector can use `vdevice#branch` format in a formula

Please see the attached files for examples using the current probe component.
[buz30a_test.zip](https://github.com/ra3xdh/qucs_s/files/10904964/buz30a_test.zip)

qucs-s version is 1.0.1 (built from Arch AUR with Qt5 option)
ngspice is version 39

**Comment by ra3xdh** [comment, marked as answer]
It is not a Qucs-S issue. Qucs-S deosn't apply any conversion to Nutmeg equations and transfers them to the simulator as is. The `i()` syntax works for TRAN simulation bot doesn't work for DC sweep. Probably Ngspice has not fully switched to new syntax.
